common/djangoapps/edxmako/apps.py

```diff
@@ -0,0 +1,22 @@
+
+from django.apps import AppConfig
+from django.conf import settings
+from . import add_lookup, clear_lookups
+
+
+class EdxMakoConfig(AppConfig):
+    name = 'edxmako'
```

Comment:
Huh, pylint isn't complaining about missing module and class docstrings?  It did when I ran it locally on a similar `apps.py` module.